### DISCUSSION TITLE
Vessel

### DISCUSSION THREAD

**Discussion by mihar-22** [vitebook/vitebook#99] [Announcements]
Decided to leave Vitebook as-is and not wreak havoc in this repo as the tooling has completely changed direction from where it originally started. Too many links and people have associated Vitebook with Storybook so best to just leave it and start fresh.

**This discussion has moved**  -> https://github.com/vessel-js/vessel/discussions/1

**Comment by oneezy** [comment]
Love it bro! Will play around with this soon

**Reply by mihar-22** [reply]
Thanks! As soon as the Beta is out you'll be able to @oneezy.

**Comment by leerob** [comment]
Let me know if you have any questions about the Build Output API 😁

**Reply by mihar-22** [reply]
Thanks @leerob, really appreciate it and will do 👍
